docs/usage/network_policies.md

```diff
@@ -0,0 +1,148 @@
+# Network Policies in Gardener
+
+As `Seed` clusters can host the `ControlPlane` of many `Shoot` clusters, it is necessary to isolate the `ControlPlanes` from each other.
+Besides deploying each `ControlPlane` in its own namespace, Gardener creates [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) to also isolate the networks. Essentially, network policies make sure that pods can only talk to other pods over the network they are supposed to.
+As such, network policies are an important part of Gardener's tenant isolation.
+
+Apart from deploying network polices into each `Shoot` namespace hosting the `ControlPlane`, Gardener also deploys policies
+ - into the namespace dedicated to Gardener system components. This namespace is often called `garden` and contains components such as the [Gardenlet](https://github.com/gardener/gardener/blob/15cae57db802cbe460ff4cb3f80c26b2fc15e26f/docs/concepts/gardenlet.md).
```

Comment:
"system components" is usally used for the `kube-system` pods in the shoot. Can we find another name? I don't have a good idea right now, maybe something like "seed-wide global controllers" or something similar.